### PR TITLE
Division by zero bug fix in metrics. Solves #102

### LINKED ISSUE #102: Division by zero in metrics module
When running the RL agent, I get division by zero error in the metrics module when using simulator run durations under 50. 

This is a new bug that came to be after #101. 

Traced issue back to `run_generated_flows` https://github.com/RealVNF/coordination-simulation/blob/89c0a489f9728e1d5b9f785d0d9344fa5a3ea6ee/src/coordsim/metrics/metrics.py#L140.

### PR DESCRIPTION
**Pull request by qarawlus**
Close #102

### CODE REVIEW

File: src/coordsim/metrics/metrics.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def reset_metrics():
 
     metrics['total_path_delay'] = 0.0
     metrics['num_path_delays'] = 0
+    metrics['run_num_path_delays'] = 0
```
Comment (stefanbschneider) [inline src/coordsim/metrics/metrics.py]: Why did you add that? Do you use it somewhere?

File: src/coordsim/metrics/metrics.py [hunk 2]
```diff
@@ -135,9 +136,8 @@ def add_path_delay(delay):
     metrics['total_path_delay'] += delay
 
     # calc path delay per run; average over num generated flows in run
+    metrics['run_num_path_delays'] += 1
     metrics['run_total_path_delay'] += delay
-    if metrics['run_processed_flows'] > 0:
```
Comment (stefanbschneider) [inline src/coordsim/metrics/metrics.py]: I'd simply adjust the if to `if metrics['run_generated_flows'] > 0:` rather than moving the code down.

**Comment by stefanbschneider** [comment]
Ready to merge from my side once the CI passes :)

**Comment by qarawlus** [comment]
Sorry for the re-request, some keyboard shortcut I guess :D
